### DISCUSSION TITLE
Security Documentation for IT

### DISCUSSION THREAD

**Discussion by MaxL777** [pycaret/pycaret#1887] [General]
Has anyone had to run PyCaret, or Python in general, past IT security to get permission to use it? If so, please share any documentation or tips you have in helping this go smoothly.

**Comment by moezali1** [comment]
@MaxL777 Little unclear what does "IT security" means? I haven't seen any library-specific security in organizations. Normally, either employees can download packages from PyPi / conda (using pip install or conda install) or they can't. It's not library-specific.
